controllers/state_of_the_world.go

```diff
@@ -26,7 +28,7 @@ import (
 
 var (
 	ConfigMapGroupKind = schema.GroupKind{Group: corev1.GroupName, Kind: "ConfigMap"}
-	operatorNamespace  = env.GetString("OPERATOR_NAMESPACE", "")
+	operatorNamespace  = env.GetString("OPERATOR_NAMESPACE", "kuadrant-system")
```

Comment:
This would make the panic at the following redundant, do we want to remove the panic now?
https://github.com/Kuadrant/kuadrant-operator/blob/abc2e19a3610e0857d30047570b24441ad90c372/controllers/state_of_the_world.go#L152-L154

Or we can remove setting a default ns env value and set it across all the makefile integration test commands that expect this env value to be set if we want to keep the panic?